### ISSUE TITLE
Install fails on 10.10.5

### ISSUE THREAD

**Issue by LucioFranco** [kyleneideck/BackgroundMusic#19]
First awesome project!

I am getting this error on the install script.

I am using os x 10.10.5.

```
Installing Background Music.app to /Applications.
Restarting coreaudiod to load BGMDriver.
Invalid signal.
Usage: launchctl kill <signal-number|signal-name> <service-target>
ERROR: Install script failed at line 200. This is probably a bug in the script. Feel free to report it.
```

**Comment by kyleneideck** [comment]
Thanks for reporting this. It should be fixed in 0178644417ead43b4df83fb331cb963f29bbde36, but since you've already installed (and restarting coreaudiod is the last thing the install script does) you should be able to just do `sudo killall coreaudiod` and then open Background Music.app.

**Comment by LucioFranco** [comment]
Seems like it worked! Will report back if I find any more issues. Thanks!
